.github/workflows/main.yml

```
@@ -0,0 +1,47 @@
+# This is a basic workflow to help you get started with Actions
+
+name: tweet_scheduler
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  push:
+    branches: [ main ]
+  #schedule:
+  #  - cron: '30 10,18 * * *'    
```

Perhaps add in the manual 'button click' dispatch as well (for easy testing where you don't need to push)
```suggestion
  push:
    branches: [ main ]
  on: workflow_dispatch
  #schedule:
  #  - cron: '30 10,18 * * *'    
```